internal/lib/cron/cron.go

```diff
@@ -63,3 +63,15 @@ func (s *ScheduleSpec) getEarliestTimeToStartCron(currTime time.Time) time.Time
 	}
 	return startTime
 }
+
+func (s *ScheduleSpec) GetExpectedRuns(startTime, endTime time.Time) []time.Time {
```

Comment:
is this included in the scope of this PR?

Reply:
yes

Reply:
removed